test/tests/misc/version/tests

```diff
@@ -0,0 +1,14 @@
+[Tests]
+  [./print_version_long]
+    type = 'RunApp'
+    input = 'IGNORED'
+    expect_out = '^Version: \S+'
```

Comment:
This seem incorrect. `Version` is now not at the beginning of the line. It should say `MooseTest Version: \S+`.

The same below.

Reply:
Right...